lib/process-manager.ts

```diff
@@ -9,25 +9,23 @@
  * @license MIT
  */
 
-'use strict';
+import * as child_process from 'child_process';
+import * as path from 'path';
+import { ObjectReadStream, ObjectReadWriteStream } from './streams';
```

Comment:
I would `import * as Streams`

Reply:
I'd quibble, but there's better things to do and youre the boss :D.